### ISSUE TITLE
Feature request: Menu item for fullscreen

### ISSUE THREAD

**Issue by bluss** [jupyterlab/jupyterlab-desktop#851]
F11 for full screen works well, when jupyterlab desktop has keyboard focus. (It doesn't work when jupyterlab's content has focus, because there is a default keybinding for F11. It can be removed, but let's assume default configuration.)

Suggestion: Add menu item for toggling full screen. For discoverability and predictable usability. F11 works if I click the menu, then use F11 (to ensure the right thing has keyboard focus).

**Comment by JasonWeill** [comment]
Related issue in JupyterLab itself: https://github.com/jupyterlab/jupyterlab/issues/8710
